File: Modul 05/03/PRAK403-2410817310020-MRizkiDinar.c
#include <stdio.h>
#include <stdlib.h>
int main()
{
    int x,y,rang;
    int modifier = 1;
    scanf("%d %d", &x,&y);
    if(y>x){
        modifier = -1;
    }
    rang = abs(x-y);
    for(int i = 0; i<=rang; i++){
        printf("%d %d", x, y);
        x -= modifier;
        y += modifier;
        if(i != rang){
            printf(" - ");}
    }
    return 0;
}

File: Modul 04/03/PRAK403-2410817310020-MRizkiDinar.py
x, y = map(int, input().split())
modifier = 1

if y > x:
    modifier = -1

rang = abs(x-y)+1
for i in range(rang):
    print(x, y, end = "")
    x -= modifier
    y += modifier
    if i != rang-1:
        print(" - ", end = "")